soc/arm64/rockchip/rk3568/soc.c

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2021 Arm Limited (or its affiliates). All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/kernel.h>
+
+#ifdef CONFIG_SOC_RK3568
+
+/*
+ * fvp can work on the mode that all core run together when it start.
+ * So an implementation for FVP is needed.
```

Comment:
What's the point with FVP here?

Reply:
Thanks for the comment, I forgot to delete it when I added FVP support earlier, I will modify it.